cypress/e2e/CMS/$SiteSections.feature

```diff
@@ -77,7 +101,8 @@ Feature: As a cms user I want to be able to create Site Section to promote Site
         And user selects "children" link under "Home"
         And user selects "children" link under "About Cancer"
         And user selects "children" link under "Coping with Cancer"
-        And user clicks on "Delete" from  dropdown button under "Test Site Section"
+        And user selects "Delete" operation for "Test Site Section"
+        #And user clicks on "Delete" from  dropdown button under "Test Site Section"
```

Comment:
is this is not needed, then remove this step)

Reply:
Removed